LayoutTests/imported/w3c/web-platform-tests/compression/decompression-bad-chunks.tentative.any.js

```diff
@@ -66,8 +68,10 @@ for (const chunk of badChunks) {
     const writer = ds.writable.getWriter();
     const writePromise = writer.write(chunk.value);
     const readPromise = reader.read();
-    await promise_rejects_js(t, TypeError, writePromise, 'write should reject');
-    await promise_rejects_js(t, TypeError, readPromise, 'read should reject');
+    await promise_rejects_js(t, TypeError, writer.close(), 'write should reject');
+    await promise_rejects_js(t, TypeError, reader.closed, 'read should reject');
+    writePromise.then(() => {}, () => {});
+    readPromise.then(() => {}, () => {});
```

Comment:
I would just do a one liner instead (maybe adding a comment about error handling):
writer.write(chunk.value).then(() => {}, () => {});
reader.read().then(() => {}, () => {});

Once we have this reader.read(), maybe we can add a reader.read() rejection, called after writer.close().
Something like `await promise_rejects_js(t, TypeError, reader.read(), 'write should reject');`

Reply:
That sounds good. Was doing a check for `reader.closed` afterwards too.